src/bp/react-botpress-components/src/ConfirmDialog.tsx

```diff
@@ -40,7 +40,7 @@ const ConfirmDialogComponent: FC<Props> = props => {
   return (
     <Dialog
       title={props.title}
-      icon="warning-sign"
+      icon='warning-sign'
```

Comment:
I think it would be preferrable to keep quotemarks in views

Reply:
Prettier me le change automatic au save pour des double 🤔 ma config tslint a pourtant le singlequote spécifié comme les autres repo

Reply:
IMO attributes should have double quotes and variable/js code should use single quotes, besides prettier keeps changing them back to double quotes 😂